.github/workflows/externalworkloads.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
         with:
           retry_on: error
           timeout_minutes: 1
-          max_attempts: 3
+          max_attempts: 10
```

Comment:
If 3+ retries are needed from time to time only, 10 seems a bit excessive no? Also, won't we hit the timeout of 20 minutes anyway before reaching 10 retries?

Reply:
timeout is per attempt https://github.com/nick-invision/retry#timeout_minutes

not sure what the right value is tbh.

Reply:
I was referring to the timeout of the job, on line 38 of the workflow.

Reply:
ah yes of course. personally i'd say we might as well retry until the 20 minute timeout until we come up with a better way to handle this.